github/issues_events_test.go

```diff
@@ -82,4 +152,24 @@ func TestIssuesService_GetEvent(t *testing.T) {
 	if !reflect.DeepEqual(event, want) {
 		t.Errorf("Issues.GetEvent returned %+v, want %+v", event, want)
 	}
+
+	// Test addOptions failure
+	_, _, err = client.Issues.GetEvent(ctx, "\n", "\n", -1)
+	if err == nil {
+		t.Error("bad options GetEvent err = nil, want error")
+	}
+
+	// Test s.client.Do failure
+	client.BaseURL.Path = "/api-v3/"
+	client.rateLimits[0].Reset.Time = time.Now().Add(10 * time.Minute)
+	got, resp, err := client.Issues.GetEvent(ctx, "o", "r", 1)
+	if got != nil {
+		t.Errorf("rate.Reset.Time > now ListRepositoryEvents = %#v, want nil", got)
+	}
+	if want := http.StatusForbidden; resp == nil || resp.Response.StatusCode != want {
+		t.Errorf("rate.Reset.Time > now ListRepositoryEvents resp = %#v, want StatusCode=%v", resp.Response, want)
+	}
+	if err == nil {
+		t.Error("rate.Reset.Time > now ListRepositoryEvents err = nil, want error")
+	}
```

Comment:
```suggestion
	if got != nil {
		t.Errorf("rate.Reset.Time > now GetEvent = %#v, want nil", got)
	}
	if want := http.StatusForbidden; resp == nil || resp.Response.StatusCode != want {
		t.Errorf("rate.Reset.Time > now GetEvent resp = %#v, want StatusCode=%v", resp.Response, want)
	}
	if err == nil {
		t.Error("rate.Reset.Time > now GetEvent err = nil, want error")
	}
```

Reply:
Ah, great catch!  Thank you, @nightlark !